src/pages/billing/CreateNewBillingProjectWizard.js

```diff
@@ -0,0 +1,323 @@
+        h(Link, {
+          ...Utils.newTabLinkProps, style: { textDecoration: 'underline', color: '#426ead' },
+          href: 'https://support.terra.bio/hc/en-us/articles/360026182251'
+        }, [
+          'Learn how to set up a Google Cloud Billing account'
+        ])
+
+    const checkbox =
+      div({ style: { width: '25%' } }, [
+        h(LabeledCheckbox, {
+          checked: verified === true,
+          onChange: async () => {
+            if (!isDone) {
+              if (!Auth.hasBillingScope()) { await authorizeAndLoadAccounts() }
+              setVerified(true)
+              next()
+            } else {
+              setActiveStep(3)
+              setAccessToAddBillingAccountUser(undefined)
+              setVerified(false)
+            }
+          }
+        }, [
+          p({ style: { ...styles.radioButtonLabel, marginLeft: '2rem', marginTop: '-1.3rem' } }, [
+            'I have verified the user has been added to my account (requires reauthentication)'
+          ])
+        ])
+      ])
+
+    const haveABillingAccount =
+      fieldset({ style: { border: 'none', margin: 0, padding: 0, display: 'flex', flexDirection: 'row', justifyContent: 'space-between' } }, [
+        legend({ style: { fontSize: 14, lineHeight: '22px', whiteSpace: 'pre-wrap', marginTop: '0.25rem', float: 'left' } },
+          [span({ style: { fontSize: 14, lineHeight: '22px', whiteSpace: 'pre-wrap' } },
+            ['Add ', span({ style: { fontWeight: 'bold' } }, 'terra-billing@terra.bio'), ' as a Billing Account User',
+              span({ style: { fontWeight: 'bold' } }, ' to your billing account.')]),
+          div({ style: { marginTop: '3rem' } }, [linkToSupport])]),
+        div({ style: { width: '25%' } }, [
+          div({ style: { display: 'flex', displayDirection: 'row' } }, [
+            h(RadioButton, {
+              text: "I don't have access to do this", name: 'permission',
+              checked: accessToAddBillingAccountUser === false,
+              disabled: !isDone && !isActive,
+              labelStyle: { ...styles.radioButtonLabel },
+              onChange: () => {
+                if (!isDone) {
+                  setAccessToAddBillingAccountUser(false)
+                } else {
+                  setActiveStep(3)
+                  setAccessToAddBillingAccountUser(false)
+                }
+              }
+            })
+          ]),
+          div({ style: { marginTop: '2rem', display: 'flex', displayDirection: 'row' } }, [
+            h(RadioButton, {
+              text: 'I have added terra-billing as a billing account user (requires reauthentication)', name: 'permission',
+              checked: accessToAddBillingAccountUser === true,
+              disabled: !isDone && !isActive,
+              labelStyle: { ...styles.radioButtonLabel },
+              onChange: async () => {
+                if (!isDone) {
+                  if (!Auth.hasBillingScope()) { await authorizeAndLoadAccounts() }
```

Comment:
Similar to "refresh", we probably want to call `authorizeAndLoadAccounts` here unconditionally to get the most recent billing accounts.